modules/installation-vsphere-machines.adoc

```diff
@@ -174,6 +174,11 @@ $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${
 **** `guestinfo.ignition.config.data`: Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type.
 **** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
 **** `disk.EnableUUID`: Specify `TRUE`.
++
+[IMPORTANT]
```

Comment:
Since this seems like something that you're asking the user to do always (not just in certain cases), is there a reason it couldn't just be added to the list above like the other parameters?

Reply:
This is a good point. Let me see how to incorporate this.

Reply:
@bergerhoffer Added the guidance to the existing steps. Please take one more look.